osu.Game.Rulesets.Osu/Skinning/Legacy/OsuLegacySkinTransformer.cs

```diff
@@ -30,6 +37,19 @@ public OsuLegacySkinTransformer(ISkinSource source)
         private void sourceChanged()
         {
             hasHitCircle = new Lazy<bool>(() => Source.GetTexture("hitcircle") != null);
+
+            spinnerStyle = new Lazy<SpinnerStyle>(() =>
```

Comment:
What's the point of this change, rather than just keeping it local as it was before?

Reply:
kinda like why `hasHitCircle` exists if I'm understanding this properly, less calls to `GetTexture(...)`. otherwise bonus counter needs to do `Source.GetTexture("spinner-background") != null || Source.GetTexture("spinner-top") != null`.